src/common/models/colors/colors.ts

```diff
@@ -49,6 +50,9 @@ function valueEquals(v1: any, v2: any): boolean {
   if (!v1 !== !v2) return false;
   if (v1.toISOString && v2.toISOString) return v1.valueOf() === v2.valueOf();
   if (isImmutableClass(v1)) return v1.equals(v2);
+  if (typeof v1 === "object" && typeof v2 === "object") {
+    return isEqual(v1.toJS ? v1.toJS() : v1, v2.toJS ? v2.toJS() : v2);
```

Comment:
What is going on here?

Reply:
If we use plywood range numbers and then compare them to pojsos hashed in the url, I always get false. 

The solution would be to deserialize pojsos into plywood objects, but this would require new version of url serializer (you need to save range number as `{ ...usualParams, type :"NUMBER_RANGE"}`)